packages/terra-tag/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added
+  * Added a new subcomponent, TagList.
+  * Added prop `onTagClick` in TagList which takes a callback function that is triggered when the tag is clicked.
```

Comment:
Since you are adding a new subcomponent as mentioned above, no need to mention a specific private prop in changelog.

Reply:
Changes Added. Thanks